src/components/menu/__internal__/submenu/submenu.spec.js

```diff
@@ -1134,6 +1143,38 @@ describe("Submenu component", () => {
       ).toBeFocused();
     });
 
+    /* This test is purely to achieve coverage for the else of the `handleKeyDown` 
+    callback function in the menu-item component */
+    it("should not call SubmenuContext.handleKeyDown", () => {
```

Comment:
Could you add a little more detail to this test description, such as it won't call the callback if the search has a value etc